tests/acceptance/features/apiSharingNg/mountOrUnmountShare.feature

```diff
@@ -86,3 +89,103 @@ Feature:  mount or unmount incoming share
       | resource      |
       | textfile0.txt |
       | folder        |
+
+
+  Scenario Outline: mounting shared resource by only one user in group shouldn't mount share for other user
+    Given user "Carol" has been created with default attributes and without skeleton files
+    And user "Alice" has disabled the auto-sync share
+    And user "Brian" has disabled the auto-sync share
+    And user "Carol" has uploaded file with content "hello world" to "/textfile0.txt"
+    And user "Carol" has created folder "FolderToShare"
+    And user "Carol" has sent the following share invitation:
+      | resource        | <resource> |
+      | space           | Personal   |
+      | sharee          | grp1       |
+      | shareType       | group      |
+      | permissionsRole | Viewer     |
+    When user "Alice" mounts share "<resource>" offered by "Carol" from "Personal" space using the Graph API
+    Then the HTTP status code should be "201"
+    And the response should contain the property "@client.synchronize" with value true
```

Comment:
why create new step since we have already `And the JSON data of the response should match` step. if the response is `json` may be use this step.